src/main.rs

```diff
@@ -264,6 +268,9 @@ fn main() {
                     args.concurrency,
                     args.allow_missing_relations_in_first_blocks,
                 ));
+
+                let view_client = indexer.client_actors().0;
+                actix::spawn(compute_circulating_supply(view_client));
```

Comment:
I would like to understand this function is not from the same file from the first sight. 

```suggestion
                actix::spawn(circulating_supply_provider::compute_circulating_supply(view_client));
```